Gems/ROS2/Code/Source/RobotImporter/Pages/PrefabMakerPage.cpp

```diff
@@ -17,6 +28,20 @@ namespace ROS2
         , m_parentImporterWidget(parent)
         , m_success(false)
     {
+        AZ::EBusAggregateResults<AZStd::unordered_map<AZStd::string, SpawnPointInfo>> allActiveSpawnPoints;
+        SpawnerRequestsBus::BroadcastResult(allActiveSpawnPoints, &SpawnerRequestsBus::Events::GetAllSpawnPointInfos);
+
+        m_spawnPointsComboBox = new QComboBox(this);
+        m_spawnPointsInfos = allActiveSpawnPoints.values;
+
+        for (auto i = 0; i < allActiveSpawnPoints.values.size(); i++)
```

Comment:
`auto` is unnecessary here. 
```suggestion
        for (int i = 0; i < allActiveSpawnPoints.values.size(); i++)
```